app/modules/encounters/models.py

```diff
@@ -244,10 +244,15 @@ def delete_cascade(self):
 
     def delete_frontend_request(self, request):
         response = {}
-        delete_individual = request.headers.get(
-            'x-allow-delete-cascade-individual', False
+        # headers are strings, so we need to make into bools
+        delete_individual = (
```

Comment:
Good catch but what I'm now thinking is that this is complex REST API stuff and should be in the resources file and the delete_frontend_request changes to taking the two bools rather than the full request